_components/button-groups/01-default.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Default button group
+parent: Button groups
+order: 01
+lead: The default button group arranges each button as a separate element with a gap between them. On mobile devices, the buttons are arranged vertically.
+---
+
+{% include code/preview.html component="button-groups" %}
+
+<section class="site-component-section">
+    {% include code/accordion.html component="button-groups" %}
+  <div class="usa-accordion usa-accordion--bordered site-accordion-docs">
+    <button class="usa-button-unstyled usa-accordion__button"
+        aria-expanded="true" aria-controls="accordion-bordered-docs">
+      Guidance
+    </button>
+    <div id="accordion-bordered-docs" aria-hidden="false" class="usa-accordion__content site-component-usage">
+      <h4>When to use the default button groups</h4>
+      <ul class="usa-content-list">
+        <li><strong>Actions have a contextual relationship.</strong> For example, the default button group can be used when a form has both a primary and alternative action.</li>
+      </ul>
+      <h4>When to consider something else</h4>
+      <ul class="usa-content-list">
+        <li><strong>If actions are not related.</strong> Consider how placement and alternative structure of unrelated actions can improve usability over placing all actions in a group.</li>
+        <li><strong>When mixing destructive and non-destructive actions.</strong> This can lead to making mistakes.</li>
+        <li><strong>Navigational links.</strong> Though buttons can be used for links, buttons in a group should relate to actions.</li>
```

Comment:
So we are recommending against using button groups for "Back / Next" navigation? What if that is the primary action -- for users to step through a linear set of content, such as a tutorial or gallery?

Reply:
@Ben-Judy-Flexion I think that's an appropriate use case… but wasn't sure how best to differentiate between the back/next, save&continue/skip, etc variety of links vs something like "view article." Do you have any suggestions?

Reply:
Hmm. What about:

**Linking to content.** Regular links, not buttons, should be used for navigating through site pages. However, buttons in a group can be used for directional navigation and actions (e.g., "Back," "Next," "Continue," "Skip," "Cancel," "Share.")

Reply:
I think I like that phase, "linking to content." I'd just modulate a little by saying buttons in button groups instead of all buttons. Totally normal for buttons to be used in directing users to a page if that is a primary called to action (USWDS even has examples of this).

Reply:
Is updating the existing Button guidance on the table? Because this is confusing to me, then.

![image](https://user-images.githubusercontent.com/56978395/79270532-faa1ef00-7e63-11ea-9d28-56893159d8e2.png)

Reply:
Existing guidance is on the table

Reply:
For clarity, I was referring to these instances
<img width="1503" alt="Screen Shot 2020-04-14 at 4 31 35 PM" src="https://user-images.githubusercontent.com/1094951/79271298-92580b00-7e6d-11ea-8139-adb8d8a7a080.png">

Reply:
I've updated the guidance here.